lib/player.rb

```diff
@@ -0,0 +1,23 @@
+require 'sinatra/base'
+
+class Player
+
+  attr_reader :name , :player_selection
+  
+  DEFAULT_OPTIONS = ["Rock", "Paper", "Scissors"]
```

Comment:
Good use of the array for Default options, allows easy change to the game at a later date